### PR TITLE
WETH Borrow Rate Update - Lido Instance

### PR DESCRIPTION
**Pull request by LucasWongC**
This PR proposes increasing the Slope1 parameter by 25bps for wETH on Lido instance.

The Lido instance wETH Slope1 is to be revised as follows:

| Asset | Parameter | Current | Proposed | Change |
| :---: | :-------: | :-----: | :------: | :----: |
| wETH  |  Slope1   |  2.50%  |  2.75%   | +0.25% |

### CODE REVIEW

File: src/20241024_AaveV3EthereumLido_WETHReserveBorrowRateUpdateLidoInstance/WETHReserveBorrowRateUpdateLidoInstance.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+| Asset | Parameter | Current | Proposed | Change |
+| :---: | :-------: | :-----: | :------: | :----: |
+| wETH  |  Slope1   |  2.50%  |  2.75%   | +0.25% |
+
+## References
+
+- Implementation: [AaveV3EthereumLido](https://github.com/bgd-labs/aave-proposals-v3/blob/main/src/20241024_AaveV3EthereumLido_WETHReserveBorrowRateUpdateLidoInstance/AaveV3EthereumLido_WETHReserveBorrowRateUpdateLidoInstance_20241024.sol)
+- Tests: [AaveV3EthereumLido](https://github.com/bgd-labs/aave-proposals-v3/blob/main/src/20241024_AaveV3EthereumLido_WETHReserveBorrowRateUpdateLidoInstance/AaveV3EthereumLido_WETHReserveBorrowRateUpdateLidoInstance_20241024.t.sol)
+- [Snapshot](Direct-to-AIP)
```
Comment (brotherlymite) [inline src/20241024_AaveV3EthereumLido_WETHReserveBorrowRateUpdateLidoInstance/WETHReserveBorrowRateUpdateLidoInstance.md:32]:
```suggestion
- Snapshot: Direct-to-AIP
```

File: src/20241024_AaveV3EthereumLido_WETHReserveBorrowRateUpdateLidoInstance/WETHReserveBorrowRateUpdateLidoInstance.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+## Simple Summary
+
+This publication proposes increasing the Slope1 parameter by 25bps for wETH on Lido instance.
+
+## Motivation
+
+After onboarding ezETH, the Lido instance is experiencing strong demand for borrowing wstETH.
+
+This proposal increases the wETH deposits rate by increasing the wETH Borrow Rate (Slope1). By increasing the wETH borrow rate, this offsets a portion of the wstETH deposit rate and passes through the yield from the yield maximising strategy to ETH depositors.
```
Comment (ianflexa) [inline src/20241024_AaveV3EthereumLido_WETHReserveBorrowRateUpdateLidoInstance/WETHReserveBorrowRateUpdateLidoInstance.md:16]:
```suggestion
This proposal increases the wETH deposit rate by increasing the wETH Borrow Rate (Slope1). Increasing the wETH borrow rate offsets a portion of the wstETH deposit rate and passes the yield from the yield-maximizing strategy to ETH depositors. 
```

File: src/20241024_AaveV3EthereumLido_WETHReserveBorrowRateUpdateLidoInstance/WETHReserveBorrowRateUpdateLidoInstance.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+This publication proposes increasing the Slope1 parameter by 25bps for wETH on Lido instance.
+
+## Motivation
+
+After onboarding ezETH, the Lido instance is experiencing strong demand for borrowing wstETH.
+
+This proposal increases the wETH deposits rate by increasing the wETH Borrow Rate (Slope1). By increasing the wETH borrow rate, this offsets a portion of the wstETH deposit rate and passes through the yield from the yield maximising strategy to ETH depositors. 
+
+The resulting increase in wETH deposit rate is expected to enable the DAO to revise lower the ETH deposit liquidity mining rewards.
```
Comment (ianflexa) [inline src/20241024_AaveV3EthereumLido_WETHReserveBorrowRateUpdateLidoInstance/WETHReserveBorrowRateUpdateLidoInstance.md:18]:
```suggestion
The resulting increase in wETH deposit rate is expected to enable the DAO to revise and lower the ETH deposit liquidity mining rewards.
```

**Review by ianflexa — APPROVED** [review]

**Comment by Matt-Graham-TL** [comment]
Agreed, lets close this PR @brotherlymite.

**Pull request closed without merging**